src/App/Security/Authorization/Voter/PrivateDiscussionVoter.php

```diff
@@ -76,26 +57,26 @@ public function vote(TokenInterface $token, $discussion, array $attributes)
         switch ($attribute) {
             case self::VIEW:
                 if ($isMember) {
-                    return VoterInterface::ACCESS_GRANTED;
+                    return true;
                 }
                 break;
             case self::EDIT:
                 if ($isAdmin) {
-                    return VoterInterface::ACCESS_GRANTED;
+                    return true;
                 }
                 break;
             case self::ADD_MEMBER:
                 if ($isAdmin and $usersNumber !== $membersNumber) {
```

Comment:
you can directly `return $isAdmin && $usersNumber !== $membersNumber;`
By the way, what is this logic? Why do you count all the users and count the discussion users?